scripts/submit_finetune_job.py

```diff
@@ -59,7 +60,11 @@ def main():
         default_yaml = f.read()
     d1 = yaml.load(default_yaml, Loader=yaml.FullLoader)
 
-    d1["tasks"][0]["image"]["beaker"] = args.image
+    if args.num_nodes > 1:
+        args.num_gpus = 8 # assume we use entire nodes if doing multinode training
```

Comment:
Can we add some sort of assert, logging, etc. to make it clear if someone tries to submit multiple nodes with < 8 gpus this is getting overriden. Probably an assert is best?

Reply:
Good call, done (with an assert). I also changed the default value of `num_gpus` to 8, so the flag can be left out if doing multinode training